### PR TITLE
Added import/order rules to auto rearrange our imports.

### PR DESCRIPTION
**Pull request by thaigillespie**
I ran 'npm run lint' but didn't really see things change. Idk if
that's just because the files I checked were already in the correct
import style, or it just didn't work... Also, had to put it in
"settings" instead of having it be a top-level "import/order" rule,
even though "eslint-plugin-import" docs on "import/order" seems
to indicate that it could be top-level.

### CODE REVIEW

**Comment by godwinpang** [comment]
I dont see it installed as a dev dep - did you run `npm install eslint-plugin-import --save-dev`?

**Comment by thaigillespie** [comment]
I see it in package.json though, that's weird.

**Comment by godwinpang** [comment]
you only pushed .eslintrc for this pr

**Comment by godwinpang** [comment]
o nvm looks like it was previously installed already

**Comment by thaigillespie** [comment]
oh yeah had to install it for the import alias resolver

**Comment by thaigillespie** [comment]
nvm lol had to put import/order in "rules" instead of "settings". It's working now.

File: src/pages/ProfileEditPage/index.js [hunk 1]
```diff
@@ -4,14 +4,16 @@ import { withStyles } from '@material-ui/core/styles';
 import { Card, CardContent, Grid } from '@material-ui/core';
 import { Formik, Form } from 'formik';
 
+import schema from './schema';
+
```
Comment (godwinpang) [inline src/pages/ProfileEditPage/index.js:8]: any way to remove this newline?

File: config-overrides.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (godwinpang) [inline config-overrides.js:1]: I think this eslint suppression line is suppressing the wrong line now :)

File: .eslintrc [hunk 3]
```diff
@@ -30,6 +30,42 @@
       {
         "ignore": ["firebase", "classes", "history", "children"]
       }
+    ],
+    "import/order": [
+      "error",
+      {
+        "newlines-between": "always-and-inside-groups",
```
Comment (godwinpang) [inline .eslintrc:37]: aha I've found the offender

Comment (thaigillespie) [reply .eslintrc:37]: I was a little conflicted with this choice, while I like newlines to separate groups of imports, I also don't want to enforce no newlines within the groups themselves so I used this, but looks like it didn't work out too well...

Comment (godwinpang) [reply .eslintrc:37]: how about always?

Comment (godwinpang) [reply .eslintrc:37]: anyhow - I trust that you have more visibility into this than I do so just pick whatever makes sense and we can move on

**Comment by thaigillespie** [comment]
I mean honestly for this it's really up to what we think is the best way to auto arrange our imports. There is no one best way imo, just whatever fits the best with everyone or at least what everyone can work with. This can always be tweaked without breaking the code so yeah. The rules I added for import/order is just how I think the imports should be arranged. This is subjective so feel free to make changes to this, now that we have a way to change the rules to import/order.

**Pull request merged**